app/views/admin/adjustments/edit.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="col-md-6 col-md-offset-3 col-sm-12 reg-page">
+  <div class="headline">
+    <h1><%= t('adjustment.edit') %></h1>
+  </div>
+  <%= render 'form' %>
+    <%= link_to t(:destroy_attr, attr: model_name(Adjustment)),
```

Comment:
I tried to use these translations and I think it ended up rather messy.
I do prefer to use `t('adjustment.destroy')`.